server/src/test/groovy/com/objectcomputing/checkins/controllers/CheckInControllerSpec.groovy

```diff
@@ -56,6 +57,7 @@ class CheckInControllerSpec extends EmbeddedServerSpecification implements Check
         cleanup:
         checkInRepository.deleteAll()
         memberProfileRepository.deleteAll()
+        skillRepository.deleteAll()
```

Comment:
Need to delete all items in skillRepository to make sure 'assert !hasLeakage()' is satisfied. Records for Skills table is inserted from V4__load_initial_skills.sql that needs to be cleared.